tests/python/unittest/test_operator.py

```diff
@@ -883,7 +883,6 @@ def test_nearest_upsampling():
                     shapes = [(1,3,base*root_scale*scale**(num_shape-1-i),base*root_scale*scale**(num_shape-1-i)) for i in range(num_shape)]
                     check_nearest_upsampling_with_shape(shapes, scale, root_scale)
 
-
```

Comment:
Did you check if all the refactored operators have their own unit test? I know some operators (e.g. `FullyConnected`) don't have unit test. We should add them to make sure the refactoring doesn't break any operator.

Reply:
FullyConnected may not have its own unit test, but it's used in others' unit tests.

Reply:
I'm just saying that we'd better do a full scan to make sure all the refactored operators are tested in some way.

Reply:
There's a C++ unit test fully_conn_perf.cc in tests/cpp although it's just performance-related for now, but value tests could be added if desired.